### PR TITLE
Integrates CircleCI for continuous integration

### LINKED ISSUE #364: Use CircleCI for continuous integration
Derived from samvera/maintenance#16

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #364.

### CODE REVIEW

**Comment by jrgriffiniii** [comment]
Coveralls is now reporting for this.

**Comment by cjcolvar** [comment]
There has been work done in the past to remove solr configs from individual gems and to centralize it in ActiveFedora so I'd be really hesitant to reintroduce it here.

**Comment by jrgriffiniii** [comment]
Understood, apologies for that, I'll look in to how best to generate the Solr. config. from `active-fedora` and POST it to the Docker container on CircleCI.

**Comment by jrgriffiniii** [comment]
This now just uses `active-fedora`

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          ruby_version: << parameters.ruby_version >>
+          bundler_version: << parameters.bundler_version >>
+          project: hydra-works
+      - run:
+          name: Wait for Solr
+          command: dockerize -wait tcp://localhost:8985 -timeout 1m
+      - run:
+          name: Load config into Solr
+          command: |
+            cd "$(bundle show active-fedora)/lib/generators/active_fedora/config/solr/templates/solr/config"
```
[the remark this answers is not in the document]
Comment (jrgriffiniii) [reply .circleci/config.yml]: This has been pushed into https://github.com/samvera-labs/samvera-circleci-orb/pull/14, and when that is acceptable and merged, I can update the Orb dependency in this.

**Pull request merged**